### PR TITLE
[Docs] Add Tutorial :Deploying PHP Guestbook application(Stateless) with Redis

### PR DESCRIPTION
**Pull request by deepak4566**
**Notes for Reviewers**
There is some remaining docs , need to be added ,will add it soon 

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e1dbf01074d45e3dc1f59c3d818c1e8a9df82957 at: https://66701969e6b7b137f80cd592--meshery-docs-preview.netlify.app

**Comment by deepak4566** [comment]
@vishalvivekm //

**Comment by l5io** [comment]
🚀 Preview for commit 0c866da9f589605c661a8ddfd824c159761fa1fb at: https://6670440aee256521b4ba5503--meshery-docs-preview.netlify.app

File: docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+### Introduction
+
+This tutorial shows you how to build and deploy a simple (not production ready), multi-tier web application using Kubernetes and Docker. This example consists of the following components:
+- A single-instance Redis to store guestbook entries
+- Multiple web frontend instance
+
+### Prerequisites:
+- Basic understanding of containerization , Kubernetes concepts and basic knowledge of reddis .
+- Access to the _Me
+- shery Playground_. If you don't have an account, sign up at [Meshery Playground](https://meshery.layer5.io/).
```
Comment (dusdjhyeon) [inline docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md:23]: Don't line break.

File: docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+list: include
+---
+### Introduction
+
+This tutorial shows you how to build and deploy a simple (not production ready), multi-tier web application using Kubernetes and Docker. This example consists of the following components:
+- A single-instance Redis to store guestbook entries
+- Multiple web frontend instance
+
+### Prerequisites:
+- Basic understanding of containerization , Kubernetes concepts and basic knowledge of reddis .
```
Comment (dusdjhyeon) [inline docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md:21]: reddis -> redis

Comment (dusdjhyeon) [reply docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md:21]: Delete the space before the comma(,) and the space before the period(.).

File: docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  # type: LoadBalancer
+  #type: LoadBalancer
+  ports:
+    # the port that this service should serve on
+  - port: 80
+  selector:
+    app: guestbook
+    tier: frontend
+```
+#### Step6 - Vizualizing  the Cassandra StatefulSet
```
Comment (dusdjhyeon) [inline docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md:223]: Vizualizing -> Visualizing

Comment (dusdjhyeon) [reply docs/pages/guides/tutorials/Deploying-Guestbook-Application-With-Redis.md:223]: Also, is the docs about Cassandra correct?

**Review by dusdjhyeon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**